parm/atm/obs/testing/amsua_metop-a.yaml

```diff
@@ -18,7 +18,7 @@ obs space:
         type: H5File
         obsfile: !ENV amsua_metop-a_diag_${CDATE}.nc4
```

Comment:
I don't think `!ENV` is the way we now specify `obsfile`.   `config/amsua_metop-a.yaml` specifies the ability `obsfile` as `$(DATA)/obs/$(OPREFIX)amsua_metop-a.`   This comment applies to other amsua yamls in `testing`

There are other occurrences of `!ENV` in the testing amsua yamls.   Do we need to change these?

Reply:
@RussTreadon-NOAA @CoryMartin-NOAA 
I fixed the following:
1.  Modified obsfile name to follow the naming convention
2. defined Sensor_ID

Reply:
I think it's okay for the testing YAMLs to include this old YAML style, especially if others in testing look like this, they are intended to be only for UFO acceptance and not "real" end to end use.